docs/source-app/glossary/storage/drive_content.rst

```diff
@@ -4,13 +4,24 @@
 About Drives
 ************
 
-Lightning Drive storage makes it easy to share files between LightningWorks so you can run your Lightning App both locally and in the cloud without changing the code.
+Lightning Drive storage makes it easy to share files between LightningWorks (Work) so you can run your Lightning App both locally and in the cloud without changing the code.
 
 The Drive object provides a central place for your components to share data.
 
-The Drive acts as an isolate folder and any component can access it by knowing its name.
+The Drive acts as an isolated folder and any component can access it by knowing its name.
 
-Your components can put, list, get, and delete files from and to the Drive (except LightningFlows).
+We currently support two types of Drives: ``lit://`` and ``s3://``.
+
++---------------+-------------------------------------------------------------------------------------------------------------------------------+
+| ``lit://``    | ``lit://`` allow read-write operations and are accessible through the Drive API from a Work.                                  |
+|               |                                                                                                                               |
+|               | They allow your components to put, list, get, and delete files from and to the Drive (except LightningFlows).                 |
++---------------+-------------------------------------------------------------------------------------------------------------------------------+
+| ``s3://``     | ``s3://`` is AWS S3 storage mounted at a filesystem mount point. ``s3://`` is read-only (for now) and its primary purpose is  |
+|               | to give you a permanent location to store your training data.                                                                 |
```

Comment:
```suggestion
| ``s3://``     | ``s3://`` is AWS S3 storage mounted at a filesystem mount point. ``s3://`` is read-only (for now) and its primary purpose is  |
|               | to give you a permanent location to access your training data.                                                                 |
```
Since earlier it said read-only, maybe avoid the word "store"?